climada/engine/unsequa/unc_output.py

```diff
@@ -1139,7 +1152,55 @@ def __init__(self, samples_df, unit, aai_agg_unc_df, freq_curve_unc_df, eai_exp_
         self.at_event_sens_df = None
         self.coord_df = coord_df
 
+class UncDeltaImpactOutput(UncOutput):
+    """Extension of UncOutput specific for CalcDeltaImpact, returned by the  uncertainty() method.
+    """
+    def __init__(self, samples_df, unit, aai_agg_unc_df, freq_curve_unc_df, eai_exp_unc_df,
+                 at_event_initial_unc_df, at_event_final_unc_df, tot_value_unc_df, coord_df):
+        """Constructor
 
+        Uncertainty output values from impact.calc for each sample
+
+        Parameters
+        ----------
+        samples_df : pandas.DataFrame
+            input parameters samples
+        unit : str
+            value unit
+        aai_agg_unc_df : pandas.DataFrame
+            Each row contains the value of aai_aag for one sample (row of
+            samples_df)
+        freq_curve_unc_df : pandas.DataFrame
+            Each row contains the values of the impact exceedence frequency
+            curve for one sample (row of samples_df)
+        eai_exp_unc_df : pandas.DataFrame
+            Each row contains the values of eai_exp for one sample (row of
+            samples_df)
+        at_event_unc_df : pandas.DataFrame
+            Each row contains the values of at_event for one sample (row of
+            samples_df)
+        tot_value_unc_df : pandas.DataFrame
+            Each row contains the value of tot_value for one sample (row of
+            samples_df)
+        coord_df : pandas.DataFrame
+            Coordinates of the exposure
```

Comment:
Why not a `geopandas.GeoSeries`? Is there any expectation about the column names, like `latitude`, `longitude`? Could you add this to the documentation?

Reply:
I just mirrored that from the `UncImpactOutput` class. You may need to consult with @chahank if you'd like to see such a change.

Reply:
I'll let that up to you and @chahank to decide and suggest you modify the docstring of CalcImpact in the same go too. I don't know what the columns must be called.

Reply:
Fine. Let's leave it as is.